build/images/tf_operator/Dockerfile

```diff
@@ -4,6 +4,8 @@ ADD . /go/src/github.com/kubeflow/tf-operator
 
 WORKDIR /go/src/github.com/kubeflow/tf-operator
 
+RUN go mod download
```

Comment:
I think we have the vendor in the git repo.

Reply:
yeap, but for common library, there is only `common/pkg/apis/common/v1` api definition in the vendor folder
but no `common/pkg/controller.v1/common` (which include `reconcileJob`, `reconcilePod`... these function)